src/plugins/saved_objects/public/services.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { createGetterSetter } from '@kbn/kibana-utils-plugin/public';
+import { SavedObjectsStart } from './plugin';
+
+export const [getSavedObjects, setSavedObjects] =
```

Comment:
Probably irrelevant given the previous comment, but we'd love to avoid module-static accessors in our owned plugins and use a more classical explicit dependency injection flow.